src/app/admin/modals/user-settings-modal/user-settings-modal.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { currentUser } from 'src/app/ajs-upgraded-providers';
+import { User, UserService } from 'src/app/api/models/doubtfire-model';
+import { DoubtfireConstants } from 'src/app/config/constants/doubtfire-constants';
+
+@Component({
+  selector: 'user-settings-modal',
```

Comment:
as the [CI](https://github.com/thoth-tech/doubtfire-web/runs/6115326969?check_suite_focus=true) mentioned : The selector should start with one of these prefixes: "df". Can we change that?